pyiron/atomistics/structure/atoms.py

```diff
@@ -1016,125 +1017,53 @@ def repeat_points(self, points, rep, centered=False):
     def reset_absolute(self, is_absolute):
         raise NotImplementedError("This function was removed!")
 
+    @deprecate("use analyse.pyscal_cna_adaptive() with ovito_compatibility=True instead.")
```

Comment:
Shouldn't it be `Atoms.analyse.pyscal_cna_adaptive()` ? I'm not so sure about the convention though...

Reply:
I an also not aware of a convention. I will switch to your version though.